modules/culling/docs/api-reference/oriented-bounding-box.md

```diff
@@ -52,6 +52,14 @@ The center position of the box.
 
 The transformation matrix, to rotate the box to the right position.
 
+### readonly halfSize: number[]
+
+The array corresponding to I3S OBB halfSize (https://github.com/Esri/i3s-spec/blob/master/docs/1.7/obb.cmn.md)
```

Comment:
Style Nit: The way this is written is a little too specific. This is supposed to be a very generic class. and the new representation is not an invention of I3S, it just happens to be used there.

I would just say that this member returns a vector with the three halfSizes for the bounding box. And that the quaternion member returns a quaternion describing the orientation of the bounding box.

 I think you could have a note in the introduction section that we support two representations for obbs and you can link to the I3S docs as an example there. around line 4 or 5.